python/pyarrow/parquet.py

```diff
@@ -1190,6 +1171,9 @@ def _make_manifest(path_or_paths, fs, pathsep='/', metadata_nthreads=1,
 memory_map : boolean, default True
     If the source is a file path, use a memory map to read file, which can
     improve performance in some environments
+read_dictionary : list, default None
+    List of column paths to read directly as DictionaryArray. Only supported
```

Comment:
The "paths" might be a bit confusing for people not familiar with that parquet terminology. Column "names" ?

Reply:
Hm. I don't think you can use Parquet and hide from this detail

Reply:
I think an example explaining this in the parquet section would already clarify a lot (and enough, I didn't meant to suggest to change the API itself).

Reply:
`column names or paths` should be a good alternative that neither hides the format details nor confuses new users.

Reply:
OK. I'll expand the docstring and give a couple examples